src/components/Data/Documents/DocumentItem.vue

```diff
@@ -14,8 +14,12 @@
     <label class="item-title"><a @click="toggleCollapse">{{document.id}}</a></label>
 
     <div class="right actions">
-      <a v-link="{name: 'DataUpdateDocument', params: {id: document.id}}"><i class="fa fa-pencil"></i></a>
-      <dropdown :id="document.id" class="icon-black">
+      <a
+        v-link="{name: 'DataUpdateDocument', params: {id: document.id}}"
+        v-if="rights.canEdit">
```

Comment:
Same as above: 
We have not to hide element, but disable it with a help text.

See exemple here: [src/components/Data/Indexes/List.vue#L19](https://github.com/kuzzleio/kuzzle-backoffice/blob/master/src/components/Data/Indexes/List.vue#L19)